### ISSUE TITLE
Larger curves

### ISSUE THREAD

**Issue by davidwengier** [davidwengier/Trains.NET#4]
Curved tracks would look a lot better if they could be bigger, and take up 2x2 cells. In general, support for larger items on cells might be useful for The Future™

**Comment by ScottHolden** [comment]
I like the idea of an abstract x*y tile that can define which edges tracks connect via. Would allow for a 3*3 y connector

**Comment by davidwengier** [comment]
Yes, and the API would be easy enough too.. perhaps an `IEnumerable<ConnectionPoint>` on a track or something.

**Comment by ScottHolden** [comment]
Yeah, in the example of the 3 * 3 y connector, would it return the 3 cells that are connected, and their direction of connection?

**Comment by davidwengier** [comment]
Maybe... I don't know because `ConnectionPoint` doesn't exist. Otherwise a `track.ConnectsTo(col, row)` would work.
